packages/server/lib/browsers/utils.ts

```diff
@@ -249,7 +249,7 @@ const getBrowsers = async () => {
     return browsers
   }
 
-  const version = process.versions.chrome || ''
+  const version = process.versions.electron || ''
```

Comment:
Hmm this seems like a pretty obvious bug, surprised it flew under the radar for so long. Will need to check this, good job calling this out

Reply:
The fix would not work without this. Otherwise, Cypress would not spin up electron browser with `ELECTRON_RUN_AS_NODE=1` because `version` would be an empty string.

Reply:
I had asked @flotwig about this before. He said had told me:

> The Chrome key there is the version of chromium bundled into Electron, we use that as the display version because it's more useful to the user since it correlates to Chrome versions. Whereas "Electron 19" to a user is less useful.

Reply:
Oh, right, neat trick. We should add a comment.

Reply:
@XiaolongLyft can you please revert this, as pointed out it's intentional.

Reply:
Edit: I reverted it for you.